.c3i/authorized_users.yml

```
@@ -1359,3 +1359,8 @@ authorized_users:
 - tttapa
 - zeeshancs07
 - jll63
+- Jie-Cheng
+- jjardina
+- soonick
+- Mercotui
```

```suggestion
- Mercotui
- sunbubble
```
https://github.com/sunbubble
https://github.com/conan-io/conan-center-index/issues/4#issuecomment-2139505358